src/applet/profile/disable.c

```diff
@@ -22,10 +21,8 @@ static int applet_main(int argc, char **argv) {
 
     param = argv[1];
 
-    refreshflag = false;
-    if (argc > 2) {
-        refreshflag = atoi(argv[2]) != 0;
-    }
+    if (argc > 2)
+        refreshflag = str_to_bool(argv[2]);
```

Comment:
Note refreshflag is a bool, so if str_to_bool returned -1, refreshflag will be true. Is this what you want?

Reply:
NO